static/app/components/events/groupingInfo/groupingVariant.tsx

```diff
@@ -159,28 +158,17 @@ function GroupingVariant({event, variant, showNonContributing}: GroupingVariantP
   const renderTitle = () => {
     const isContributing = variant.hash !== null;
 
-    let title: string;
-    if (isContributing) {
-      title = t('Contributing variant');
-    } else {
-      const hint = 'component' in variant ? variant.component?.hint : undefined;
-      if (hint) {
-        title = t('Non-contributing variant: %s', hint);
-      } else {
-        title = t('Non-contributing variant');
-      }
-    }
+    const hint = 'component' in variant ? variant.component?.hint : undefined;
 
     return (
-      <Tooltip title={title}>
-        <VariantTitle>
-          <ContributionIcon isContributing={isContributing} />
-          {variant.description
-            ?.split(' ')
-            .map(i => capitalize(i))
-            .join(' ') ?? t('Nothing')}
-        </VariantTitle>
-      </Tooltip>
+      <VariantTitle>
+        <ContributionIcon isContributing={isContributing} />
+        {variant.description
+          ?.split(' ')
+          .map(i => capitalize(i))
+          .join(' ') ?? t('Nothing')}
+        <VariantHint>{!isContributing && hint && t('(%s)', hint)}</VariantHint>
```

Comment:
```suggestion
        <VariantHint>{hint && t('(%s)', hint)}</VariantHint>
```

or

```suggestion
        <VariantHint>{hint && t(`(${hint})`}</VariantHint>
```

For now it's true that only non-contributing variants get hints, but I don't think there's any reason to lock ourselves into that. Might as well keep the code as simple as we can, too. (If you do pull that condition out, don't forget to update the PR title, too.)

The template-literal part is totally a stylistic choice. I think it's a little more readable, but completely up to you.